co-log-core/src/Colog/Core/Action.hs

```diff
@@ -7,6 +7,8 @@
 module Colog.Core.Action
        ( -- * Core type and instances
          LogAction (..)
+        , (<&)
+        , (&>)
```

Comment:
Here you have 8 spaces, while it should be 7 according to this module code style

Reply:
Hmm, perhaps we should look into adding a stylish haskell rule for this if we can?

Reply:
Unfortunately, `stylish-haskell` doesn't have this configurations :(

Reply:
Yep, here is an open issue:

* https://github.com/jaspervdj/stylish-haskell/issues/220